WooCommerce/Classes/POS/Models/PointOfSalePaymentState.swift

```diff
@@ -49,7 +51,9 @@ extension PointOfSalePaymentState {
         switch self {
         case .processingPayment,
                 .paymentError,
-                .cardPaymentSuccessful:
+                .cardPaymentSuccessful,
+                .acceptingCash,
+                .cashPaymentSuccessful:
```

Comment:
Good question. Some of the presentation logic for the rest of the collection flow checks if `.cardPaymentSuccessful`, which won't make much sense as state when the order is paid successfully via cash. Since moving the cash view presentation to not rely on payment state I think right now we can get rid of `.acceptingCash`, but we still need to do something with `.cardPaymentSuccessful`. 

I'll check both into using a new state, or as you mention making this a `.paymentSuccessful` state with associated value for cash or card 👍